### DISCUSSION TITLE
Logging In on iPhone

### DISCUSSION THREAD

**Discussion by dustyr** [lukevella/rallly#202] [Q&A]
Is there a way to stay logged in on an iPhone/iPad?  Users complain they have to sign in every time they access the website. Appreciate any suggestions you may have.

**Comment by lukevella** [comment]
Yes it should work. Perhaps they are using Private browsing in safari? This would log you out when you close the tab.

**Comment by dustyr** [comment]
The couple that complained said they were using Chrome but I tried it with
Chrome and it seems to work OK.  I believe that the issue is that they set
up a Shortcut to the website on their phones and it looks like that creates
a whole new chrome session each time which is why they have to log in.  I
just tested this and can confirm that the issue was with the Shortcut.  I
converted them to using a bookmark in Chrome instead and it works fine.
Thank you for the prompt response.

**Reply by lukevella** [reply]
Oh I see. I guess to fix this I have to move away from magic links and use a code instead which you can type in. That would allow you to use shortcuts like this. Will add that to the backlog. Thanks for mentioning.
